src/common/utils.py

```diff
@@ -121,3 +121,7 @@ def is_url_contains_a_token(url) -> bool:
 
 def str_to_bool(s: str) -> bool:
     return bool(int(s))
+
+
+def raw_str_to_bool(s: str) -> bool:
+    return True if s == "true" else False
```

Comment:
return s == "true"

Reply:
I am not sure this function is required, you call it only twice in the code and the command is very simple.

Reply:
I think even if it is only two times, it should be a different function because the name of the function helps to understand why we are even doing this.